main.go

```diff
@@ -220,7 +215,11 @@ func main() {
 		logFail("Performing request failed, error: %#v", requestErr)
 	}
 
-	defer response.Body.Close()
+	defer func() {
+		if err := response.Body.Close(); err != nil {
+			logWarn("Failed to close response body:", err)
+		}
+	}()
 	contents, readErr := ioutil.ReadAll(response.Body)
 	if readErr != nil {
 		logWarn("Failed to read response body, error: %#v", readErr)
```

Comment:
logWarn("Failed to read response body, error: %#v", readErr)
->
logWarn("Failed to read response body, error: %s", readErr)